### DISCUSSION TITLE
See the actually installed APK ?

### DISCUSSION THREAD

**Discussion by Ugau943** [ImranR98/Obtainium#1414] [Q&A] [answered]
Hi,

Maybe I've missed something somewhere, but I can't find if there's a way to see which APK has been installed by Obtainium, i.e. which APK has been selected in the source.

In this case I'm filtering an APK by regex (on GitHub, where there's an F-droid version and another), but not being absolutely sure of the reliability of my filter, I wanted to make sure of the name of the APK that has actually been downloaded and installed.

So is this information somewhere in Obtainium ?

**Comment by DwainZwerg** [comment, marked as answer]
> So is this information somewhere in Obtainium ?

No. Maybe you want to create an issue?

**Comment by Ugau943** [comment]
Yep, it's done. Thank you ! 

#1426

**Reply by inson1** [reply]
@Ugau943 btw dont forget to close it, when you did fr from it :)
